app/src/main/java/com/example/navadon/androidnamecard/FrontActivity.java

```diff
@@ -19,6 +22,13 @@ protected void onCreate(Bundle savedInstanceState) {
     private void initView() {
         initOnClickListener();
 
+        FrontActivityViewModel frontActivityViewModel = new FrontActivityViewModel();
+
+        ActivityFrontBinding activityFrontBinding = DataBindingUtil.setContentView(this, R.layout.activity_front);
+        activityFrontBinding.setModel(frontActivityViewModel);
+
+        frontActivityViewModel.setFrontflip("tap B to flip");
+
         findViewById(R.id.imageB).setOnClickListener(onClickListener);
```

Comment:
Actually, you can do something like this...
activityFrontBinding.imageB.setOnClickListener(onClickListener);
No need to use findViewById().